packages/core/src/button/inline-button.element.ts

```diff
@@ -50,14 +49,6 @@ export class CdsInlineButton extends CdsBaseButton {
   }
 
   static get styles() {
-    return [baseStyles, baseButtonStyles, styles];
```

Comment:
Realized that removing the base button styles simplified the styles. Looks like base button styles are for buttons with our block style button